### PR TITLE
103 chaning support phone number

### LINKED ISSUE #103: chaning support phone number
changing the support phone number to the right one

### PR DESCRIPTION
**Pull request by BuchiNy**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #[issue number] 

**Checklist**
- [ ] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [ ] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [ ] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [ ] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginScreen.kt [hunk 1]
```diff
@@ -347,6 +357,7 @@ fun LoginPage(
 fun ForgotPasswordDialog(
   forgotPassword: () -> Unit,
   onDismissDialog: () -> Unit,
+  applicationConfiguration: ApplicationConfiguration,
```
Comment (sevenreup) [inline android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginScreen.kt]: Pass the actual phone number here, not ApplicationConfiguration

File: android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginScreen.kt [hunk 2]
```diff
@@ -98,6 +99,10 @@ fun LoginScreen(loginViewModel: LoginViewModel) {
   val password by loginViewModel.password.observeAsState("")
   val loginErrorState by loginViewModel.loginErrorState.observeAsState(null)
   val showProgressBar by loginViewModel.showProgressBar.observeAsState(false)
+  val applicationConfiguration by
+    loginViewModel.applicationConfiguration.observeAsState(
+      ApplicationConfiguration(supportPhoneNumber = ""),
```
Comment (sevenreup) [inline android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginScreen.kt]: You really don't need to init the support number to an empty string, since it already has a default

**Review by sevenreup — CHANGES_REQUESTED** [review]

**Review by sevenreup — APPROVED** [review]
